src/sources.cmake

```diff
@@ -190,20 +190,20 @@ target_sources(precice
     src/mesh/Filter.hpp
     src/mesh/Mesh.cpp
     src/mesh/Mesh.hpp
-    src/mesh/Quad.cpp
-    src/mesh/Quad.hpp
     src/mesh/RTree.cpp
     src/mesh/RTree.hpp
     src/mesh/RangeAccessor.hpp
     src/mesh/SharedPointer.hpp
     src/mesh/Triangle.cpp
     src/mesh/Triangle.hpp
+    src/mesh/Utils.hpp
     src/mesh/Vertex.cpp
     src/mesh/Vertex.hpp
     src/mesh/config/DataConfiguration.cpp
     src/mesh/config/DataConfiguration.hpp
     src/mesh/config/MeshConfiguration.cpp
     src/mesh/config/MeshConfiguration.hpp
+    src/mesh/impl/BBUtils.hpp
```

Comment:
Why is this (only) added here? Why did it not give problems earlier?

Reply:
CMake only requires compilation units / cpp files to be listed. The compiler is responsible for finding the right headers.
Listing them will make them appear in IDEs though, so it is generally recommended.